packages/framework/src/Resources/styles/admin/core/form/custom-inputs.less

```diff
@@ -33,7 +35,7 @@ input[type=radio].css-radio + .css-radio__image, input[type=radio].css-radio + .
 		content: "";
 		position: absolute;
 		display: inline-block;
-		top: -9px;
+		top: 0;

```

Comment:
this change is because of the set dimensions?

Reply:
i think this is because Selenia can´t find element. These less changes are ok for me.

Reply:
Yes, because I've set the height to 18px, I don't need to set negative `top`.